xml/System.Text/EncodingProvider.xml

```diff
@@ -368,17 +368,17 @@ This method is called by the [Encoding.GetEncoding(String)](<xref:System.Text.En
         <summary>Returns the encoding associated with the specified name. Parameters specify an error handler for characters that cannot be encoded and byte sequences that cannot be decoded.</summary>
         <returns>The encoding that is associated with the specified name, or <see langword="null" /> if this <see cref="T:System.Text.EncodingProvider" /> cannot return a valid encoding that corresponds to <paramref name="name" />.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
```

Comment:
Note: I wanted to let you know that all these extra trailing spaces will get automatically added again by the build system. I've also tried removing them in the past but it's no use.